toolkit/tools/scheduler/scheduler.go

```diff
@@ -65,6 +66,7 @@ var (
 	distroBuildNumber    = app.Flag("distro-build-number", "The distro build number that the SRPM will be built with.").Required().String()
 	rpmmacrosFile        = app.Flag("rpmmacros-file", "Optional file path to an rpmmacros file for rpmbuild to use.").ExistingFile()
 	buildAttempts        = app.Flag("build-attempts", "Sets the number of times to try building a package.").Default(defaultBuildAttempts).Int()
+	checkAttempts        = app.Flag("check-attempts", "Sets the minimum number of times to test a package.").Default(defaultCheckAttempts).Int()
```

Comment:
Let's rephrase this is a bit to be clear that the retries only happen in case of a test failures.